app/services/streaming/streaming-view.ts

```diff
@@ -97,39 +151,50 @@ export class StreamInfoView extends ViewHandler<IStreamingServiceState> {
     return getPlatformService(this.userView.auth.primaryPlatform).chatUrl;
   }
 
+  getTweetText(streamTitle: string) {
+    return `${streamTitle} ${Services.TwitterService.url}`;
```

Comment:
One of the "rules" of views is that they shouldn't inject services.  The proper way to access other services from within a view handler is `this.getServiceViews()` which restricts your access to the views on another service, ensuring you don't accidentally do any IPC.